src/webview/components/kdbDataSourceView.ts

```diff
@@ -217,7 +217,7 @@ export class KdbDataSourceView extends LitElement {
     if (this.isInsights && this.isMetaLoaded) {
       return this.insightsMeta.api
         .filter(
-          (api) => api.api === ".kxi.getData" || !api.api.startsWith(".kxi.")
+          (api) => api.api === ".kxi.getData" //|| !api.api.startsWith(".kxi.")
```

Comment:
what is the response from getMeta ?
where are the other apis? there is one bug related to missing customapi, this probably should be part of that ticket

Reply:
The problem is discussed on https://kxsys.slack.com/archives/C057T0J659N/p1702391947279529 and the ticket is opened. We do not support any other api than `getData ` at the moment.  If we decide to support custom apis, we need to design api screens according to that custom API.

Reply:
If I miss something, we can discuss on slack

Reply:
In that case, could you take SS of this thread and add to the ticket. 
Also create an ticket with story and tag this actual ticket to it, to implement dynamic custom api for the future, sor we don't lose the track and don't have issues in the future when someone say: "I don't remember it, I didn't said that", just to keep the records